crucible-mir/src/Mir/Trans.hs

```diff
@@ -1880,6 +1880,7 @@ initLocals localVars addrTaken = forM_ localVars $ \v -> do
         Just (MirExp tpr' val) -> do
             Refl <- testEqualityOrFail tpr tpr' $
                 "initialValue produced " ++ show tpr' ++ " instead of " ++ show tpr
+                  ++ ", for " ++ show ty
```

Comment:
Slightly more compact:

```suggestion
                  ++ ", for " ++ show (pretty ty)
```

Reply:
Fixed in 579afd3857198af9fc108870f387483beb58c677